From 3d9a93334e0fd76af3e60a1e907083160dc1c0ea Mon Sep 17 00:00:00 2001
From: thongsyho <thong.ho.px@rvc.renesas.com>
Date: Tue, 9 Jan 2018 19:48:33 +0700
Subject: [PATCH 251/641] ARM: DTS: RZG1C r8a77470: Add Video Signal Processing
 module (vspm module) node to device tree

This patch support Renesas vspm kernel module probe.

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
---
 arch/arm/boot/dts/r8a77470.dtsi | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/r8a77470.dtsi b/arch/arm/boot/dts/r8a77470.dtsi
index 8741ee3..98b3b88 100644
--- a/arch/arm/boot/dts/r8a77470.dtsi
+++ b/arch/arm/boot/dts/r8a77470.dtsi
@@ -507,6 +507,22 @@
 		renesas,#ch = <0>;
 	};
 
+	vspm@fe928000 {
+		compatible = "renesas,vspm-vsps", "renesas-vspm";
+		reg = <0 0xfe928000 0 0x7404>;
+		interrupts = <0 267 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&mstp1_clks R8A77470_CLK_VSP1_S>;
+		power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
+	};
+
+	vspm@fe930000 {
+		compatible = "renesas,vspm-vspd0", "renesas-vspm";
+		reg = <0 0xfe930000 0 0x7404>;
+		interrupts = <0 246 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&mstp1_clks R8A77470_CLK_VSP1_DU0>;
+		power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
+	};
+
 	fdp0: fdpm@fe940000 {
 		compatible = "renesas,fdp1";
 		reg = <0 0xfe940000 0 0x2400>;
-- 
2.7.4

